src/components/HistoryDialog/index.tsx

```diff
@@ -208,53 +293,105 @@ const HistoryDialog = <T extends string>({
+            <Grid item xs={3}>
+              <StyledEventItem
+                color={color}
+                textAlign="left"
+                data-testid={`history-item-${index}-status`}
+              >
+                {status?.toUpperCase()}
+              </StyledEventItem>
+            </Grid>
+            <Grid item xs={3}>
+              <StyledEventItem
+                color={color}
+                title={date}
+                data-testid={`history-item-${index}-date`}
+              >
+                {FormatDate(date, "M/D/YYYY", "N/A")}
+              </StyledEventItem>
+            </Grid>
+            {eventHasNames && (
+              <Grid item xs={3} data-testid={`history-item-${index}-name`}>
+                <StyledEventItem>
+                  <TruncatedText
+                    text={name}
+                    maxCharacters={14}
+                    wrapperStyles={{
+                      ...BaseItemTypographyStyles,
+                      margin: "0 auto",
+                      color: nameColor,
+                    }}
+                  />
+                </StyledEventItem>
+              </Grid>
+            )}
+            <Grid
+              item
+              xs={1}
+              sx={{
+                position: "relative",
+                marginLeft: eventHasNames ? "0" : "auto",
+                marginRight: eventHasNames ? "0" : "24px",
+              }}
             >
-              <StyledTimelineSeparator>
-                <StyledTimelineDot />
-                <StyledTimelineVerticalLine />
-              </StyledTimelineSeparator>
-              <StyledTimelineContent>
-                <Stack direction="row" alignContent="center" spacing={1} paddingRight={0}>
-                  <StyledTypography
-                    title={dateTime}
-                    color={typeof getTextColor === "function" ? getTextColor(status) : "#FFF"}
-                    data-testid={`history-item-${index}-date`}
-                  >
-                    {FormatDate(dateTime, "M/D/YYYY", "N/A")}
-                  </StyledTypography>
-                  <StyledTypography
-                    color={typeof getTextColor === "function" ? getTextColor(status) : "#FFF"}
-                    data-testid={`history-item-${index}-status`}
-                  >
-                    {status?.toString()?.toUpperCase()}
-                  </StyledTypography>
-                  {index === 0 && iconMap && iconMap[status] && (
-                    <StyledAvatar>
-                      <img
-                        src={iconMap[status]}
-                        alt={`${status} icon`}
-                        data-testid={`history-item-${index}-icon`}
-                      />
-                    </StyledAvatar>
-                  )}
-                </Stack>
-              </StyledTimelineContent>
-            </StyledTimelineItem>
-          ))}
-        </Timeline>
+              {icon && (
+                <StyledIcon>
```

Comment:
The icon seems a little outside the bounds compared to the design for Data Submissions. The Submission Request dialog looks fine.

Current:
![Screenshot 2024-10-23 at 10 53 27 AM](https://github.com/user-attachments/assets/54b57404-149f-4fd1-b162-89232c596e6d)

UI Design:
![Screenshot 2024-10-23 at 10 53 39 AM](https://github.com/user-attachments/assets/7a2d3ce3-4b79-4b00-9e40-6d480a0cafb2)